### PR TITLE
[ Desktop / Mobile ] Create SfInput

### PR DESCRIPTION
**Pull request by runyasak**
# Related issue
This PR is related to #92 

# Scope of work
Implement SfInput as atom component, the single form input component.

# Screenshots of visual changes
Alt Text

# Checklist
- [ ] I followed [composition rules](https://github.com/DivanteLtd/storefront-ui/blob/master/docs/component-rules.md) for my component
- [ ] I tested the component in most common device sizes (can be tested in Storybook [from viewport addon in top menu](https://github.com/storybooks/storybook/tree/master/addons/viewport))

### CODE REVIEW

File: src/components/atoms/SfInput/SfInput.vue [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<script src="./SfInput.js"></script>
```
Comment (filrak) [inline src/components/atoms/SfInput/SfInput.vue]: don't forget to remove it ;)

**Comment by filrak** [comment]
@runyasak how is the progress on this?

**Comment by runyasak** [comment]
@filrak Almost finish, I'm fixing css glitch on label of input. I hope that to finish with storybooks by this week. 😁

**Comment by filrak** [comment]
Thank you! For the second time I can say you did a stunning piece of work!

**Pull request merged**